doc/reference/data.rst

```diff
@@ -267,6 +267,14 @@ Mostly just proxies attributes to the corresponding fields on
 ``ir.actions.report``, but also automatically creates the item in the
 :guilabel:`More` menu of the report's ``model``.
 
+.. note::
+
+    You might expect the `name` of the `report` tag to become the `ir.actions.report` name,
+    but the values is used as `report_name` field value.  To specify the `name` field, you should use
```

Comment:
"the value**s** is used as" -> "the value is used as the"

In "To specify the `name` field, you should use the `string` attribute", I would be explicit about which object is being referred to, because I had to reread it a few times before I got it :-)

For example: "To specify the `name` field in `ir.actions.report`, you should use the `string` attribute of the `report` tag."